interfaces/mount/backend.go

```diff
@@ -121,3 +121,17 @@ func deriveContent(spec *Specification, snapInfo *snap.Info) map[string]*osutil.
 func (b *Backend) NewSpecification() interfaces.Specification {
 	return &Specification{}
 }
+
+// SandboxTags returns the list of features supported by snapd for composing mount namespaces.
+func (b *Backend) SandboxTags() []string {
+	return []string{
+		"freezer-cgroup-v1",       /* Snapd creates a freezer cgroup (v1) for each snap */
```

Comment:
Looks to me it would be nice to have a forum post listing all tags, each with a short comment on its meaning (and possibly what is required to have it available, though that only concerns apparmor and seccomp tags).

Reply:
I have now documented all of the debug commands: https://forum.snapcraft.io/t/internal-debugging-commands/5341

Reply:
Nice, thank you!